CONTRIBUTING.md

```diff
@@ -117,3 +117,18 @@ bazel test //src/python/deepgnn/graph_engine/snark/tests:python_test --test_outp
 ```
 
 This command will run all tests with prefix `test_sanity_neighbors_index`.
+
+# Create PR
+
+Follow these steps to create a pull request from a forked repo:
+1. Sync your repo with `https://github.com/microsoft/DeepGNN`.
```

Comment:
A lot of this is duplicated with https://github.com/microsoft/DeepGNN/blob/main/.github/PULL_REQUEST_TEMPLATE

Reply:
1. Only the first line is similar, everything else is different.
2. The template will appear after PR is supposed to be ready, it doesn't show what to do to create it.